### ISSUE TITLE
Error building Uno WASM project on Linux

### ISSUE THREAD

**Issue by TrevorDArcyEvans** [unoplatform/Uno.Wasm.Bootstrap#195]
[MyApp.zip](https://github.com/unoplatform/Uno.Wasm.Bootstrap/files/4528760/MyApp.zip)
Follow instructions for a sample project at:
https://platform.uno/blog/announcing-uno-platform-support-for-visual-studio-code/

Updated project to latest versions:
Uno.UI Version="2.3.0-dev.11" 
Uno.UI.RemoteControl Version="2.3.0-dev.11"
Uno.Wasm.Bootstrap Version="1.2.0-dev.26"
Uno.Wasm.Bootstrap.DevServer Version="1.2.0-dev.26"

Build output on Linux:

```bash
~/dev/MyApp $ dotnet build "/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj"
Microsoft (R) Build Engine version 16.5.0+d4cbfca49 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restore completed in 470.93 ms for /home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj.
  MyApp.Wasm -> /home/trevorde/dev/MyApp/MyApp.Wasm/bin/Debug/netstandard2.0/MyApp.Wasm.dll
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error : System.ComponentModel.Win32Exception (1
3): Permission denied [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error :    at System.Diagnostics.Process.ForkAn
dExecProcess(String filename, String[] argv, String[] envp, String cwd, Boolean redirectStdin, Boolean redirectStdout, Boolean redirectStderr, Boolean 
setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32& stdinFd, Int32& stdoutFd, Int32& stderrFd, Boolean usesTerminal, Boolean throwOn
NoExec) [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error :    at System.Diagnostics.Process.StartC
ore(ProcessStartInfo startInfo) [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error :    at System.Diagnostics.Process.Start(
) [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error :    at Uno.Wasm.Bootstrap.ShellTask_v4fd
6cf698646d7468ae0ef1f40cf3898bc0d4d77.RunProcess(String executable, String parameters, String workingDirectory) in D:\a\1\s\src\Uno.Wasm.Bootstrap\Shel
lTask.cs:line 426 [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error :    at Uno.Wasm.Bootstrap.ShellTask_v4fd6cf698646d7468ae0ef1f40cf3898bc0d4d77.RunPackager() in D:\a\1\s\src\Uno.Wasm.Bootstrap\ShellTask.cs:line 641 [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error :    at Uno.Wasm.Bootstrap.ShellTask_v4fd6cf698646d7468ae0ef1f40cf3898bc0d4d77.Execute() in D:\a\1\s\src\Uno.Wasm.Bootstrap\ShellTask.cs:line 166 [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]

Build FAILED.

/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error : System.ComponentModel.Win32Exception (13): Permission denied [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error :    at System.Diagnostics.Process.ForkAndExecProcess(String filename, String[] argv, String[] envp, String cwd, Boolean redirectStdin, Boolean redirectStdout, Boolean redirectStderr, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32& stdinFd, Int32& stdoutFd, Int32& stderrFd, Boolean usesTerminal, Boolean throwOnNoExec) [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error :    at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo) [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error :    at System.Diagnostics.Process.Start() [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error :    at Uno.Wasm.Bootstrap.ShellTask_v4fd6cf698646d7468ae0ef1f40cf3898bc0d4d77.RunProcess(String executable, String parameters, String workingDirectory) in D:\a\1\s\src\Uno.Wasm.Bootstrap\ShellTask.cs:line 426 [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error :    at Uno.Wasm.Bootstrap.ShellTask_v4fd6cf698646d7468ae0ef1f40cf3898bc0d4d77.RunPackager() in D:\a\1\s\src\Uno.Wasm.Bootstrap\ShellTask.cs:line 641 [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
/home/trevorde/.nuget/packages/uno.wasm.bootstrap/1.2.0-dev.26/build/Uno.Wasm.Bootstrap.targets(124,5): error :    at Uno.Wasm.Bootstrap.ShellTask_v4fd6cf698646d7468ae0ef1f40cf3898bc0d4d77.Execute() in D:\a\1\s\src\Uno.Wasm.Bootstrap\ShellTask.cs:line 166 [/home/trevorde/dev/MyApp/MyApp.Wasm/MyApp.Wasm.csproj]
    0 Warning(s)
    1 Error(s)
```

**Comment by jeromelaban** [comment]
Thanks for the report @TrevorDArcyEvans !

This was just fixed by #196 and #194. It should soon be published on nuget.

**Comment by jeromelaban** [comment]
On second thought, no it won't be. I'll be issuing a different fix for this one.

**Comment by jeromelaban** [comment]
Fixed by #197
